docs/ABOUT.md

```diff
@@ -12,6 +12,7 @@ You should learn JavaScript because:
 
 * It's easy to learn.
 * It's versatile in the sense that it's multi-paradigm supporting procedural, event based, object oriented and functional programming.
+* It can be used for the frontend and backend.
```

Comment:
Great addition, but I think it might deserve a new pull request. I would make a new pull request just with this line.

It doesn't matter if the PR is small.